android-client-sdk/src/main/java/com/devcycle/sdk/android/eventsource/ConnectionHandler.java

```diff
@@ -1,8 +1,7 @@
 package com.devcycle.sdk.android.eventsource;
 
-import java.time.Duration;
 
 interface ConnectionHandler {
-  void setReconnectionTime(Duration reconnectionTime);
+  void setReconnectionTime(long reconnectionTime);
```

Comment:
Does this have to be milliseconds now?
https://github.com/launchdarkly/okhttp-eventsource/compare/2.7.1...3.0.0#diff-9278fdbc0631a66e7cbc3b330a7e0d1d05f33a5590592c9f2754c72e5d762943R4

Reply:
its out own connectionHandler and can be called whatever we want, I will add a comment that says it is Milliseconds